lib/china_regions/helpers/form_helper.rb

```diff
@@ -4,9 +4,9 @@ module Helpers
     module FormHelper
       def region_select(object_name, methods, options = {}, html_options = {})
         output = ''
-        preselected_choices = {}
+        @preselected_choices = {}
```

Comment:
I am always wary of using instance variables that aren't attributes of a data-backed model - they are a shortcut that prevent proper encapsulation of data through scope. It looks like you're doing this just to avoid passing two variables to `set_preselected_choices` - why do you want to avoid this?